### PR TITLE
Install Null provider in IBM image

### PR DESCRIPTION
**Pull request by Mallear**
Had to install a few ibmcloud cli plugins and install Null provider for terraform

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-docker_hub_namespace: &docker_hub_namespace ekino
+docker_hub_namespace: &docker_hub_namespace mallear
```
Comment (clemstoquart) [inline config.yml]: Typo :)

Comment (Mallear) [reply config.yml]: Fixed :)

Comment (clemstoquart) [reply config.yml]: Thanks

**Comment by clemstoquart** [comment]
@Mallear is it ready for merge ?

**Comment by Mallear** [comment]
> @Mallear is it ready for merge ?

Totally, I've installed everything I need :)

**Pull request merged**